quic/s2n-quic-core/src/recovery/sent_packets.rs

```diff
@@ -66,20 +75,24 @@ mod test {
         frame::ack_elicitation::AckElicitation,
         inet::ExplicitCongestionNotification,
         path,
-        recovery::SentPacketInfo,
+        recovery::{bandwidth, SentPacketInfo},
         time::{Clock, NoopClock},
     };
 
     #[test]
     #[should_panic]
```

Comment:
```suggestion
    #[should_panic(expected = "sent_bytes exceeds max UDP payload size")]
```

nit: since there are more than one debug asserts in that function can you make this more specific